features/step_definitions/phut_steps.rb

```diff
@@ -5,9 +5,9 @@
   @configuration_file = file_name
 end
 
-Then(/^a vswitch named "(.*?)" launches$/) do |_name|
+Then(/^a vswitch named "(.*?)" launches$/) do |name|
   in_current_dir do
-    pid_file = File.join(Phut.settings['PID_DIR'], 'open_vswitch.0xabc.pid')
+    pid_file = File.join(Phut.settings['PID_DIR'], "open_vswitch.#{name}.pid")
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.